README.md

```diff
@@ -157,12 +173,12 @@ func application(application: UIApplication, openURL url: NSURL, sourceApplicati
 }
 ```
 
-The deep link handler is called every single time the app is opened, returning deep link data if the user tapped on a link that led to this app open.
```

Comment:
"App Open" is a proper usage here, you're not saying "that lead to your app being opened," it's saying "that lead to this instance of the app opening."